androguard/core/bytecodes/dvm.py

```diff
@@ -168,9 +168,17 @@ def get_type(atype, size=None):
 
 BREAK_DVM_OPCODES = ["invoke.", "move.", ".put", "if."]
 
-BRANCH_DVM_OPCODES = ["throw", "throw.", "if.", "goto", "goto.", "return",
-                      "return.", "packed-switch$", "sparse-switch$"]
-
+BRANCH_DVM_OPCODES = {
+    "throw" : set([0x27]),
```

Comment:
there is no need to write it like this, I guess.
you can simply write `{0x27}`, which is the same as `set([0x27])`